core/store/migrations/migrate_test.go

```diff
@@ -302,7 +311,7 @@ func TestMigrate_Migration1570675883(t *testing.T) {
 
 		require.NoError(t, migrations.MigrateTo(db, "1570675883"))
 
-		jobRunFound := models.JobRun{}
+		jobRunFound := jobRunWithOverrides{}
```

Comment:
Another option could be moving the JobRun declaration into the migration this is testing.  Which would be like 'migration1570675883.JobRun'.

What do you think?

Reply:
Yes that is better